CONTRIBUTING.md

```diff
@@ -1,13 +1,20 @@
 Contributing
 -------------
 
-Before you contribute code to PHP\_CodeSniffer, please make sure it conforms to the PHPCS coding standard and that the PHP\_CodeSniffer unit tests still pass. The easiest way to contribute is to work on a checkout of the repository, or your own fork, rather than an installed PEAR version. If you do this, you can run the following commands to check if everything is ready to submit:
+Before you contribute code to PHP\_CodeSniffer, please make sure it conforms to the PHPCS
+coding standard and that the PHP\_CodeSniffer unit tests still pass. The easiest way to
+contribute is to work on a checkout of the repository, or your own fork, rather than an
+installed PEAR version. If you do this, you can run the following commands to check if
+everything is ready to submit:
 
     cd PHP_CodeSniffer
-    php scripts/phpcs
+    composer install
```

Comment:
I don't, I can add a section making clear that this is applicable for Github workflow, e.g. git clone, composer install, change, push, PR triggers CI, PR gets review etc. I think that is quite common, but I don't have to press this on someone, it was more that the previous info was outdated.